apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureActivity.tsx

```diff
@@ -50,10 +50,18 @@ const InfrastructureActivity = () => {
   const selectedAddons = addons?.selected_addons ?? []
 
   const { computeInstance } = getAddons(selectedAddons)
-  const currentComputeInstanceSpecs =
-    computeInstance?.variant?.meta ?? project?.infra_compute_size === 'nano'
-      ? INSTANCE_NANO_SPECS
-      : INSTANCE_MICRO_SPECS
+
+  function getCurrentComputeInstanceSpecs() {
```

Comment:
Should we rather do useMemo?

Reply:
No need to IMO, this function is not expensive.